php/pantheon/checks/plugins.php

```diff
@@ -37,42 +37,52 @@ public function run() {
 		$all_plugins = Utils::sanitize_data( get_plugins() );
 		$update = Utils::sanitize_data( get_plugin_updates() );
 		$report = array();
+		$should_check_vulnerabilities = $this->getWpScanApiToken();
```

Comment:
I don't love this method returning false or string. prefer to return null or a falsy (empty) string and rename variable to reflect its usage of the token